cpp/ql/src/Likely Bugs/RedundantNullCheckSimple.ql

```diff
@@ -12,8 +12,8 @@
  */
 
 /*
- * Note: this query is not assigned a precision yet because we don't want it on
- * LGTM until its performance is well understood.
+ * Note: this query is not assigned a precision yet because we don't want it 
+ * included in query suites until its performance is well understood.
```

Comment:
```suggestion
 * Note: this query is not assigned a precision yet because we don't want it
 * included in query suites until its performance is well understood.
```
I think this trailing space was bothering the format check.